client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo/ContactDetails.tsx

```diff
@@ -16,10 +16,14 @@ const ContactDetails = (props: Props) => {
     const { errors } = useFormContext();
     const { index , interactedContact } = props;
 
+    const isStatusCompleted = interactedContact.contactStatus === 5;
```

Comment:
change the 5 value to const

Reply:
I solved this case in the schema so I removed it...